### PR TITLE
Individual skill page

### PR DESCRIPTION
**Pull request by Nadeen123**
#10 
Individual skill component
Individual skill styled component
edit data file using html to react

### CODE REVIEW

File: client/src/components/IndividualSkill/IndividualSkillStyledComponent.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  }
+`
+
+const Skilldiv = styled.div`
+  margin: 16px;
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+
+  @media only screen and (min-width: 800px) {
```
Comment (FarahZaqout) [inline client/src/components/IndividualSkill/IndividualSkillStyledComponent.js]: empty media query

File: client/src/components/IndividualSkill/IndividualSkillStyledComponent.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+const Type = styled.p`
+  text-align: center;
+  font-size: 15px;
+  color: silver;
+`
+
+const SkillImg = styled.img`
+  align-items: center;
+  @media only screen and (min-width: 800px) {
```
Comment (FarahZaqout) [inline client/src/components/IndividualSkill/IndividualSkillStyledComponent.js]: empty media query

File: client/src/components/IndividualSkill/index.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  Header,
+  Text,
+  Skilldiv,
+  TextDiv,
+  Type,
+  SkillImg,
+  List
+} from "./IndividualSkillStyledComponent"
+
+class IndividualSkill extends React.Component {
```
Comment (FarahZaqout) [inline client/src/components/IndividualSkill/index.js]: we don't need classes if we don't have a state.

File: client/src/components/IndividualSkill/IndividualSkillStyledComponent.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+  @media only screen and (min-width: 800px) {
+  }
+`
+
+const Header = styled.h1`
+  font-size: 25px;
+  color: black;
+  text-align: center;
+  padding-top: 29px;
```
Comment (FarahZaqout) [inline client/src/components/IndividualSkill/IndividualSkillStyledComponent.js]: This is too much padding. It looks okay with no padding at all.

**Comment by Nadeen123** [comment]
@FarahZaqout resolved comments

**Pull request merged**